content/how.njk

```diff
@@ -0,0 +1,125 @@
+  The typical artefacts produced at this stage would be:
+</p>
+
+<ul>
+  <li>
+    <strong>Decision logs:</strong>
+    Our most frequently produced documentation, underpinning our <a href="#decision-making">technical decision making</a>.
+  </li>
+  <li>
+    <strong>Technical documentation:</strong>
+    This is mostly applicable at the beginning of engagements for teams that do not already have architecture diagrams or a documented data model.
+    This should be considered a living document in our typical work it is often produced from the Decision Logs.
+  </li>
+  <li>
+    <strong>Risk register:</strong>
+    Once we had a chance to get a deeper understanding (by for example producing the technical documentation discussed above and trying automated workflows) we may produce a risk register with our findings.
+    While a living document, the initial version is usually produced at the beginning of an engagement.
+  </li>
+</ul>
+
+<h4 id="delivery">Delivery</h4>
+
+<p>
+  In this phase we deliver the work agreed in the Discovery phase to the standards defined in <a href="#dod">Definition of done</a>.
+</p>
+
+<p>
+  While we aim to break down work, where Discovery should have uncovered unknowns, our agile practices will allow us to change course in the face of changing requirements or priorities or uncovering "unknown unknowns".
+</p>
+
+<h4 id="reflection">Reflection and planning</h4>
+
+<p>
+  Due to our agile practices, demonstrations and gathering feedback happens continuously, however this final phase allows for a deeper assessment of the work delivered.
+</p>
+
+<p>
+  It is also when any future work is planned and agreed upon, that can kick off a new Discovery phase.
+</p>
+
+<h3 id="decision-making">Decision making</h3>
+
+<p>
+  Decisions are made daily, about architecture, implementation, future direction, technology choices, etc.
+  We believe that capturing these in a lightweight, but somewhat formalised format has benefits, not just in enabling better decisions, but also as a reference for the future and new team members.
+  Hence, we use Decision Logs, which is an iteration on <a href="https://agilemanifesto.org/principles.html">Requests for comment (RFCs)</a> and <a href="https://docs.aws.amazon.com/prescriptive-guidance/latest/architectural-decision-records/adr-process.html">Architectural decision records (ADRs)</a>.
+  A typical decision log will capture:
+</p>
+
+<ul>
+  <li>
+    <strong>Context:</strong>
+    This captures what the decision is about and why it is important.
+    It also defines the criteria for the decision.
+  </li>
+  <li>
+    <strong>Options:</strong>
+    Decisions are usually an appraisal of different options.
+    These could range from various technology choices and approaches to binary choices like whether something needs to be done or not.
+    This section captures these options with some assessment based on the criteria defined in "Context".
```

Comment:
**suggestion:**

```suggestion
    What are the options to decide between?
    This could range from architecture and technology choices to binary choices such as whether something needs to be done or not.
    This section records the evaluation of the available options against the criteria defined in "Context".
```